### PR TITLE
PMM-7 update license to Apache 2.0

### LINKED ISSUE #578: License Inconsistency in code files
The license applied to the repository is Apache2.0 as mentioned on the top level. Still, there are a lot of files under the exporter/ directory that have a license header saying they are AGPL licenses. 

Is this intentional or is that a miss when updating the license header of all the files? If that is a miss I would be happy to get create PR for updating the license.

### PR DESCRIPTION
**Pull request by BupycHuk**
Fixes #578 

- [ ] Links to other linked pull requests (optional).

---

- [ ] Tests passed.
- [ ] Fix conflicts with target branch.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

Once all checks pass and the code is ready for review, please add `pmm-review-exporters` team as the reviewer. That would assign people from the review team automatically. Report any issues on our [Forum](https://forums.percona.com).

### CODE REVIEW

File: internal/proto/proto.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package proto
```
Comment (atymchuk) [inline internal/proto/proto.go:1]: @BupycHuk Are these changes related to the license?

**Comment by atymchuk** [comment]
I believe the code needs to be formatted to make the pipelines pass (ex https://github.com/percona/mongodb_exporter/actions/runs/5410267787/jobs/9831480503?pr=679)

File: internal/proto/proto.go (hunk 1 above)
Comment (BupycHuk) [reply internal/proto/proto.go:1]: Hi, yes, we are getting rid of dependency from percona-toolkit, which has an GPL license

**Comment by atymchuk** [comment]
I believe we should remove the references to old GH actions in the settings, they will never run :)

Screenshot 2023-06-29 at 12 11 48

**Review by atymchuk — APPROVED** [review]

**Review by JiriCtvrtka — APPROVED** [review]

File: exporter/top_collector_test.go [hunk 2]
```diff
@@ -42,10 +41,10 @@ func TestTopCollector(t *testing.T) {
 	// 1. The result is huge
 	// 2. We need to check against know values. Don't use metrics that return counters like uptime
 	//    or counters like the number of transactions because they won't return a known value to compare
-	filter := []string{
-		"mongodb_top_update_count",
-	}
-	filter = nil
+	//filter := []string{
```
Comment (artemgavrilov) [inline exporter/top_collector_test.go:44]: Leftover?

**Review by artemgavrilov — APPROVED** [review]

File: exporter/top_collector_test.go (hunk 2 above)
Comment (BupycHuk) [reply exporter/top_collector_test.go:44]: I have no idea what happened there, so I decided to leave it as a comment to make linters not complain about that.
